operator/v1/types_kubeapiserver.go

```diff
@@ -35,6 +35,18 @@ type KubeAPIServer struct {
 
 type KubeAPIServerSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+
+	// eventTTLMinutes specifies the amount of time that the events are stored before being deleted.
+	// This setting is allowed between 30 minutes minimum up to 6h (360 minutes).
```

Comment:
Because the field value is an integer, it would be better to reverse these
```suggestion
	// The TTL is allowed between 30 minutes minimum up to a maximum of 360 minutes (6 hours).
```